tmt/_compat/importlib/metadata.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import annotations
```

Comment:
This seems to be appearing in PRs. We should discuss whether we want/need this first before adding it in a unrelated PR.